frontendVue3/src/components/ConstraintAddEditDialog.vue

```diff
@@ -223,6 +217,8 @@ export default {
                 this.constraintText.slice(0, pos) +
                 textToInsert +
                 this.constraintText.slice(pos);
+            console.log("New constraintstext: "+ this.constraintText);
+            this.selectedFeatureNode=undefined;
```

Comment:
After hours of debugging this non logic Line fixed an issue with the v-model i did not understand.

Need some assistance here :smile: